pyaerocom/aeroval/setupclasses.py

```diff
@@ -163,6 +163,10 @@ class StatisticsSetup(BaseModel, extra="allow"):
 
     """
 
+    # Pydantic ConfigDict
+    model_config = ConfigDict()
+    model_config["protected_namespaces"] = ()
```

Comment:
The example on the [pydantic docs] shows the `protected_namespaces` set on the object instantiation. For our case this would be equivalent to:

``` python
    model_config = ConfigDict(protected_namespaces=())
```

[pydantic docs]: https://docs.pydantic.dev/latest/api/config/#pydantic.config.ConfigDict.protected_namespaces

Reply:
Sure, we can do it like that. I changed the occurrence of this in `WebDisplaySetup` as well.